tests/phpt/spread/001_spread_ok.php

```diff
@@ -0,0 +1,49 @@
+@ok php7_4
```

Comment:
Do we need `php7_4` tag?

Reply:
Until all tests run on 7.4, we need this tag here.